cmd/bootstrap/run/epochs.go

```diff
@@ -174,8 +176,12 @@ func GenerateRecoverTxArgsWithDKG(
 		}
 	}
 
+	csprg, err := prg.New(epoch.RandomSource(), prg.BootstrapClusterAssignment, nil)
```

Comment:
nit-pick : although the PRG instantiated here is indeed a CSPRG, this case does not require the "CS" part. 
The reason is that the seed is public and we're not intending to hide it or hide the future outputs of the PRG. We need a crypto secure PRG when the seed is confidential and future outputs should not be derived from past outputs.
For this reason, a use of PRG is enough in all our bootstrap cases in this PR, and we can simply use the word `prg`. Using the word `csprg` implies that crypto-security is needed.

```suggestion
	prg, err := prg.New(epoch.RandomSource(), prg.BootstrapClusterAssignment, nil)
```